cves/CVE-2019-9721.yml

```diff
@@ -90,22 +95,24 @@ fixes_vcc_instructions: |
   find interesting in there.
 fixes:
 - commit: 894995c41e0795c7a44f81adc4838dedc3932e65
-  note: ''
-- commit: ''
-  note: ''
+  note: |
+    due to the complex nature of these file inputs, this vulnerability is not completely resolved.
+    authored by: Kevin Vackhouse via RT
+    signed-off-by: Michael Niedermayer
 vccs:
 - commit: 50332f8a6bc303b6147fe34a481a2c0b80ac12be
-  note: Identified by archeogit,Identified by SZZUnleashed
-- commit: ''
-  note: ''
+  note: |
+    Introudced the htmlsubtitles.c file which the vulnerability exited in.
```

Comment:
Please refrain from mentioning specific files. In addition, please leave the fact that these commits were identified by tools.